extensions/pkg/controller/reconciler.go

```diff
@@ -81,6 +81,7 @@ func (o *operationAnnotationWrapper) Reconcile(request reconcile.Request) (recon
 	if annotations[v1beta1constants.GardenerOperation] == v1beta1constants.GardenerOperationReconcile {
 		withOpAnnotation := obj.DeepCopyObject()
 		delete(annotations, v1beta1constants.GardenerOperation)
+		delete(annotations, v1beta1constants.GardenerReconciliationRequest)
```

Comment:
Can you remove it again? We didn't want to make it an official contract that extensions "have" to follow. The Gardenlet "just" adds the annotation, but it isn't needed to be removed again. Also, I think now that we have the timestamp annotation it could make operations  somewhat easier in some cases.

Reply:
ok, I see, will do.